### DISCUSSION TITLE
IOSender Tool Orientation

### DISCUSSION THREAD

**Discussion by RJ5X** [terjeio/ioSender#376] [General]
Hi Terjeio, 

Can the tool orientation be shown in IOSender? 
I can see the tool-pad and actual tool position but not the tool orientation.

Thanks,

**Comment by terjeio** [comment]
What do you mean by tool orientation? B-axis position? Or lathe tool orientation? Or something else?

**Reply by RJ5X** [reply]
Hi terjeio,

Thanks for your attention.

I am working on the creation of a five-axis head table machine. 
I am using rotational axis 'A' and 'C'. 

![5AXIS_HEAD_TABLE_Definition](https://github.com/terjeio/ioSender/assets/165228027/1b798107-c475-444c-b675-cc1133a68dc4)

I would like to know. It is possible to show the tool angle at the graphic, versus the position of the 'X', 'Y', and 'Z' axes in IOSender.

![Capture](https://github.com/terjeio/ioSender/assets/165228027/4f22f11d-a42f-4f35-85ad-45d22b2b272a)


Thanks,

**Reply by RJ5X** [reply]
Sorry if my question does not make sense. 
I am modifying 'grbl6axis.cps' for a 5-Axis Head-Table machine as shown in the previous images. The point is, I would like to see if I am doing the right thing. 
The tool can be projected from any point in the space, even from the inside of this sphere in the picture to the 'X', 'Y', and 'Z' axis positions, where the tool is supposed to land.
The question is: Could the tool's virtual angle be actively shown in 'IOSender'?
If that is possible, it will be a nice option to include, It will help a lot for development.

![Tool_Angle](https://github.com/terjeio/ioSender/assets/165228027/b2f11b70-c592-4c93-8691-8fb1acb5e6e5)


 I appreciate the time you are using to support this community.

Thanks,

**Reply by terjeio** [reply]
> The question is: Could the tool's virtual angle be actively shown in 'IOSender'?

Possibly, I'll have to check.
